src/Lykke.Logs/Console/LykkeConsoleLogger.cs

```diff
@@ -0,0 +1,271 @@
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                Console = new WindowsLogConsole();
+            }
+            else
+            {
+                Console = new AnsiLogConsole(new AnsiSystemConsole());
+            }
+        }
+
+        public IConsole Console
+        {
+            get => _queueProcessor.Console;
+            set => _queueProcessor.Console = value ?? throw new ArgumentNullException(nameof(value));
+        }
+
+        public Func<string, Microsoft.Extensions.Logging.LogLevel, bool> Filter
+        {
+            get => _filter;
+            set => _filter = value ?? throw new ArgumentNullException(nameof(value));
+        }
+
+        public bool IncludeScopes { get; set; }
+
+        public string Name { get; }
+
+        public void Log<TState>(Microsoft.Extensions.Logging.LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter)
+        {
+            if (!IsEnabled(logLevel))
+            {
+                return;
+            }
+
+            var callerInfo = state as LogEntryParameters;
+            if (callerInfo == null)
+            {
+                throw new ArgumentNullException(nameof(state), "Expected an argument state with a type assignable to LogEntryParameters");
+            }
+
+            if (formatter == null)
+            {
+                throw new ArgumentNullException(nameof(formatter));
+            }
+
+            var message = GetExceptionString(exception) ?? formatter(state, exception);
+
+            if (!string.IsNullOrEmpty(message) || exception != null)
+            {
+                WriteMessage(logLevel, callerInfo, Name, message);
+            }
+        }
+
+        private static string GetExceptionString(Exception exception)
+        {
+            return exception?.ToAsyncString();
+        }
+
+        private void WriteMessage(Microsoft.Extensions.Logging.LogLevel logLevel, LogEntryParameters callerInfo, string logName, string message)
+        {
+            var logBuilder = _logBuilder;
+            _logBuilder = null;
+
+            if (logBuilder == null)
+            {
+                logBuilder = new StringBuilder();
+            }
+
+            // Example:
+            // INFO: ConsoleApp.Program[10]
+            //       Request received
+
+            var logLevelColors = GetLogLevelConsoleColors(logLevel);
+            var logLevelString = GetLogLevelString(logLevel);
+            // category and event id
+            logBuilder.AppendFormat(@"{0:yyyy-MM-dd HH:mm:ss:fff}", callerInfo.Moment);
+            logBuilder.Append(" [");
+            logBuilder.Append(logLevelString);
+            logBuilder.Append("] ");
+            logBuilder.Append(logName);
+            logBuilder.Append(":");
+            logBuilder.Append(callerInfo.Process);
+            logBuilder.Append(":");
+            logBuilder.Append(callerInfo.Context);
```

Comment:
Use Lykke.Common.Log.LogContextConversion.ConvertToString() to convert context to the string

Reply:
ok